doc/tutorial/02_index/index.md

```diff
@@ -1,19 +1,274 @@
+It can also be used for metagenomic classification, i.e., which microbes are in a single sample.
 
-`raptor build --kmer 19 --window 23 --compute-minimiser --output precomputed_minimisers all_bin_paths.txt`
+\see Related tools:
+- Rna-seq expression analysis: [Needle](https://www.seqan.de/apps/needle.html), a tool for storing sequencing
+experiments in such a way that approximate quantification of large sequencing data sets is possible.
+- Metagenomics: [Ganon/Ganon2](https://github.com/pirovc/ganon), a tool for comparative metagenomics analysis in short
+time using the whole of the quickly growing number of assembled sequences openly available.
 
-`raptor build --size 8m --output minimiser_raptor.index all_minimiser_paths.txt`
+Regardless of the application, we start with a data set of different nucleotide sequences over which we want to build an
+index.
+
+These sequences are typically available as
+[FASTA](https://docs.seqan.de/seqan/3-master-user/classseqan3_1_1format__fasta.html#a86790afd92e0229cccbc20be20d5758a) or
+[FASTQ](https://docs.seqan.de/seqan/3-master-user/classseqan3_1_1format__fastq.html#details) files, but can be in any
+format supported by
+[`seqan3::sequence_file_input`](https://docs.seqan.de/seqan/3-master-user/tutorial_sequence_file.html).
+
+\assignment{Assignment 1: Create example files and index them}
+Copy and paste the following FASTA files to some location, e.g. the tmp directory:
+
+mini_1.fasta:
+```fasta
+>chr1
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACGCGTTCATTAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+```
+mini_2.fasta:
+```fasta
+>chr2
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACGCGTCATTAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+```
+mini_3.fasta:
+```fasta
+>chr3
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+```
+and then create an all_paths.txt and run raptor build.
+\endassignment
+
+\solution
+Your all_paths.txt should look like:
+```txt
+mini_1.fasta
+mini_2.fasta
+mini_3.fasta
+```
+And you should have run:
+```bash
+raptor build --output raptor.index all_paths.txt
+```
+Your directory should look like this:
+```bash
+tmp$ ls
+all_paths.txt   mini_1.fasta    mini_2.fasta    mini_3.fasta    raptor.index
+```
+\endsolution
+\todo Currently there is an Error: `[Error] Option --size is required but not set.` Thus you should run raptor with the
+default size value: `--size 1k`.
+
+\note
+We will use this mini-example in the following, both with further parameters and then for `raptor search`. We therefore
+recommend not deleting the files including the built indexes.
+
+We summarise these in a list of paths and can then create a first index using the default values of raptor:
+```bash
+raptor build --output raptor.index all_bin_paths.txt
+```
+
+\note
+Raptor also has a help page, which can be accessed as usual by typing `raptor build -h` or `raptor build --help`.
```

Comment:
This block (L79-L89) should maybe be in front of the assignment? It resolves some comments I had above.

Reply:
I would leave the first note after the assignment, because I find it more intuitive to solve the task first and then read that I should not delete the files afterwards. But I have put it in the solution part.
I have sorted the rest at the top. Is that OK?